api/http/handler/webhooks/handler.go

```diff
@@ -0,0 +1,38 @@
+package webhooks
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	portainer "github.com/portainer/portainer"
+	httperror "github.com/portainer/portainer/http/error"
+	"github.com/portainer/portainer/http/security"
+)
+
+// Handler is the HTTP handler used to handle webhook operations.
+type Handler struct {
+	*mux.Router
+	WebhookService   portainer.WebhookService
+	EndpointService  portainer.EndpointService
+	requestBouncer   *security.RequestBouncer
+	SignatureService portainer.DigitalSignatureService
+}
+
+// NewHandler creates a handler to manage settings operations.
+func NewHandler(bouncer *security.RequestBouncer) *Handler {
+	h := &Handler{
+		Router:         mux.NewRouter(),
+		requestBouncer: bouncer,
+	}
+	h.Handle("/webhooks",
+		bouncer.RestrictedAccess(httperror.LoggerHandler(h.webhookCreate))).Methods(http.MethodPost)
+	h.Handle("/webhook/{serviceID}",
```

Comment:
I'd like to see the following changes on the API:

* Remove the `webhookInspect` handler as it's not used correctly. It is actually used to retrieve a webhook associated to a specific service identifier but should not be used this way. For consistency, the inspect method should use a webhook identifier as a route parameter (but there is no need to inspect a webhook atm, so just remove it).

* Update the `webhookList` handler to support a `filters` query parameter, this will allow you to search for a webhook associated to a service identifier. If you need an example you can have a look at: https://github.com/portainer/portainer/blob/develop/api/http/handler/stacks/stack_list.go#L19

* Replace the parameter of the `webhookDelete` handler with `{id}` to keep a consistent API. You'll need to delete the webhook by it's identifier rather than by referencing the service identifier.